impeller/renderer/command.cc

```diff
@@ -58,11 +58,11 @@ bool Command::DoBindResource(ShaderStage stage,
   switch (stage) {
     case ShaderStage::kVertex:
       vertex_bindings.buffers[slot.ext_res_0] = {
-          .slot = slot, .view = BufferResource(metadata, view)};
+          .slot = slot, .view = BufferResource(metadata, std::move(view))};
```

Comment:
If I'm reading the code correctly, this used to be a copy here, but now is a move which should be faster.